src/lib/operators/get_table.cpp

```diff
@@ -47,7 +50,16 @@ std::string GetTable::description(DescriptionMode description_mode) const {
   stream << AbstractOperator::description(description_mode) << separator;
   stream << "(" << table_name() << ")" << separator;
   stream << "pruned:" << separator;
-  stream << _pruned_chunk_ids.size() << "/" << stored_table->chunk_count() << " chunk(s)";
+  auto overall_pruned_chunk_ids = _dynamically_pruned_chunk_ids;
+  overall_pruned_chunk_ids.insert(_pruned_chunk_ids.begin(), _pruned_chunk_ids.end());
```

Comment:
I don't think that it matters here, but the copy and the insert should resize the vector "perfectly" (more or less) to the size of the input vectors. With the `.insert()` (which should correctly add the space for `_pruned_chunk_ids`) that leads to a second copy of `_dynamically_pruned_chunk_ids`.
First creating `overall_pruned_chunk_ids`, reserving it to the sum of both vectors' sizes, and then using `.insert()` twice, should be faster here.

Reply:
You might have been aware of it. I was not, so I thought I share this super interesting insight of the year.

And yes ... it's just the description. Doesn't matter.